internal/grpc/commander/connection_test.go

```diff
@@ -172,33 +172,74 @@ func TestConnection_HandleCommand(t *testing.T) {
 }
 
 func TestConnection_HandleAgentConnectRequest(t *testing.T) {
-	g := NewGomegaWithT(t)
 
-	out := make(chan *proto.Command)
+	invalidConnectRequest := &proto.Command{
+		Meta: &proto.Metadata{
+			MessageId: "msg-id",
+		},
+		Data: &proto.Command_AgentConnectRequest{
+			AgentConnectRequest: &proto.AgentConnectRequest{
+				Meta:    &proto.AgentMeta{},
+				Details: []*proto.NginxDetails{},
+			},
+		},
+	}
 
-	fakeExchanger := &exchangerfakes.FakeCommandExchanger{
-		OutStub: func() chan<- *proto.Command {
-			return out
+	tests := []struct {
+		name          string
+		request       *proto.Command
+		expStatusCode proto.AgentConnectStatus_StatusCode
+		expStatusMsg  string
+	}{
+		{
+			name:          "normal",
+			request:       CreateAgentConnectRequestCmd("msg-id"),
+			expStatusCode: proto.AgentConnectStatus_CONNECT_OK,
+			expStatusMsg:  "Connected",
+		},
+		{
+			name:          "invalid",
+			request:       invalidConnectRequest,
+			expStatusCode: proto.AgentConnectStatus_CONNECT_REJECTED_OTHER,
+			expStatusMsg:  "missing nginxID: \"\" and/or systemID: \"\"",
```

Comment:
would it make sense to use `...` for readability? to avoid `\"\"`

Reply:
`...`?  I don't understand the suggestion

Reply:
lol, I forgot to check how the message renders

I meant:
\`string\`